lib/src/flow.cpp

```diff
@@ -31,6 +32,21 @@ mxlStatus mxlCreateFlow(mxlInstance instance, char const* flowDef, char const* /
         }
         return MXL_ERR_INVALID_ARG;
     }
+    catch (std::filesystem::filesystem_error const& e)
+    {
+        MXL_ERROR("Failed to create flow : {}", e.what());
+        auto code = e.code();
+        if (code == std::errc::permission_denied || code == std::errc::operation_not_permitted || code == std::errc::read_only_file_system)
```

Comment:
I'm a big fan to put separate sets of parentheses around binary sub-expressions, as I think it makes the code more readable and less prone to wonky precedence conflicts.